### PR TITLE
v1.1.0 - Support for primus-multiplex

### PR DESCRIPTION
**Pull request by tranquilo**
Hi there. 
Added support for primus-multiplex.

Cheers,
Ricardo Cunha

### CODE REVIEW

File: angular-primus.js [hunk 1]
```diff
@@ -119,6 +119,46 @@ function primusProvider() {
...
+          var args = arguments;
+          $rootScope.$evalAsync(function () {
+            listener.apply(null, args);
+          });
+        }
+
+        // Return the deregistration function
+        return function $off() {
+          channel.removeListener(event, listenerInAngularContext);
+        };
```
Comment (Yvem) [inline angular-primus.js]: we ought to start factorizing this code...

**Comment by Yvem** [comment]
lack_of_testing

**Comment by tranquilo** [comment]
lollolol
I'm sorry but unfortunately i'm really busy with several projects and couldn't afford to put all the cheese in it.
I had to make my custom provider for this but i thought it would be nice to share my code for the multiplex functionality.

**Comment by neoziro** [comment]
@tranquilo OK, we can refactor your PR, thanks.

**Pull request closed without merging**